src/components/VocabularyListItem.tsx

```diff
@@ -79,17 +78,19 @@ const VocabularyListItem = ({ document }: VocabularyListItemPropType) => {
     <View style={styles.wrapper}>
       <View style={styles.container}>
         <View style={styles.item}>
-          <Image
-            source={{
-              uri: document.document_image[0].image
-            }}
-            width={24}
-            height={24}
-            style={styles.image}
-          />
+          {document.document_image.length > 0 && (
+            <Image
+              source={{
+                uri: document?.document_image[0].image
+              }}
+              width={24}
+              height={24}
+              style={styles.image}
+            />
+          )}
           <View>
             <Text testID='article' style={[styles.title, { backgroundColor: getArticleColor(article) }]}>
-              {article?.toLowerCase() === ARTICLES.diePlural ? 'Die' : capitalizeFirstLetter(article)}
+              {article.value}
```

Comment:
Previously this was capitalized:
```suggestion
              {capitalizeFirstLetter(article.value)}
```

Reply:
Ich habe jetzt in data.ts einfach mal die ersten Buchstaben groß gemacht, dann können wir uns das hier sparen und ich denke wir brauchen es nie in klein.